pyhap/accessories/SDS011.py

```diff
@@ -73,20 +73,20 @@ def _set_services(self):
         # PM2.5
         air_quality_pm25 = loader.get_serv_loader().get("AirQualitySensor")
         pm25_size = char_loader.get("AirParticulateSize")
-        pm25_size.set_value(0, should_notify=False)
+        pm25_size.value = 0
```

Comment:
I though that accessories should be calling `set_value`, and in the future `value` will be made read only with a `@property` getter.

Reply:
I didn't mentioned that above, but I realized that there are cases in which setting the value on its one does make sense. Testing as well as the one above. `value = 0` is basically equivalent to `set_value(0, should_notify=False)` if no callback has been added and the char doesn't belong to any service / accessory yet.

Reply:
Yes, but doing this will ignore all the validation logic, which still needs to be run at some point. This could allow an invalid value to be added using `__init__` and then sent to an iOS client during a `get`.

Better to just use `set_value`, and have the char figure out if sending a notification makes sense (if there is no broker, then just don't bother with the notification).